cartofacade/carto_facade.go

```diff
@@ -173,3 +173,126 @@ func (cf *CartoFacade) start(ctx context.Context, activeBackgroundWorkers *sync.
 		}
 	}()
 }
+
+// Initialize calls into the cartofacade C code.
+func (cf *CartoFacade) Initialize(ctx context.Context, timeout time.Duration, activeBackgroundWorkers *sync.WaitGroup) error {
```

Comment:
Question: Do we need to take the `activeBackgroundWorkers` as a parameter here or can we just create one within this function?

Reply:
I believe we want to take it in because we want to call `ActiveBackgroundWorkers.Wait()` outside of this function because if we initialize it here  and pass it in, and defer the call to  `ActiveBackgroundWorkers.Wait()`, it will block the function from returning